src/test/resources/org/sonatype/nexus/ci/config/ComToOrgMigratorIntegrationTest/jobs/Freestyle-IQ/config.xml

```diff
@@ -32,7 +32,9 @@
           <scanPattern>target/*.jar</scanPattern>
         </com.sonatype.nexus.ci.iq.ScanPattern>
       </com__sonatype__nexus__ci__iq__IqPolicyEvaluator____iqScanPatterns>
-      <com__sonatype__nexus__ci__iq__IqPolicyEvaluator____iqApplication>sample-app</com__sonatype__nexus__ci__iq__IqPolicyEvaluator____iqApplication>
+      <applicationSelectTypePost>select</applicationSelectTypePost>
```

Comment:
This is testing the com to org package migration. There is no implementation of this plugin with com packages where these fields are included so this is not required.